integration_tests/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+docker-compose up -d
+docker cp integration_tests/data/ mysterium-client:/usr/lib/integration_tests
```

Comment:
Does it have to be copied into container?

Reply:
That's strange to copy something into running container at all

Reply:
Yes - tequilapi binds on localhost, so it is not reachable from host machine.

Reply:
Yes, it is reachable:
- map 4050 to wanted host port 
- in wort case, You can bind on other unix socket (see container environment variables https://github.com/MysteriumNetwork/node/blob/master/bin/client_package/installation/default)

Reply:
It's ok to change it to "0.0.0.0" for testing purposes. At least that's why this parameter --tequilapi.address "0.0.0.0" was intended for ;)

Reply:
Testing environment should as as close to real as possible, but I can't find any better solution to keep that, so we'll have to make it public in tests :/